### PR TITLE
Add storybook

### PR DESCRIPTION
**Pull request by nath-green**
Packages and configuration for storybook.

Searches `../src/core` for any `story.js` and compiles them.

Each component will need a story file going forward, a separate PR will be created for existing components 💯

To run: `yarn storybook`

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "import/prefer-default-export": false,
     "no-underscore-dangle": 0,
     "react/prop-types": false,
-    "jsx-a11y/label-has-for": false
+    "jsx-a11y/label-has-for": false,
```
Comment (aorv) [inline .eslintrc]: what's this puppy doing?

File: .storybook/config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { configure } from '@storybook/react';
+import '!style-loader!css-loader!sass-loader!../src/styles/main.scss';
```
Comment (aorv) [inline .storybook/config.js]: `!style-loader!css-loader!sass-loader!..`
never seen this in an import before - is this parcel related?

File: .eslintrc (hunk 1 above)
Comment (nath-green) [reply .eslintrc]: Its only showing that as a changed lined because I added a `,` to add the line below 👇

That `has-label-for` is for the `Input` atom so that you don't have to wrap the input in a label but can reference the input and label with an ID 👍

Then the `jsx-filename-extension` is because the story files are `.js` but they include `jsx` to render out the component

File: .storybook/config.js (hunk 2 above)
Comment (nath-green) [reply .storybook/config.js]: It's a webpack thing man, a shortcut way to tell webpack what modules to use when importing that file otherwise its expecting a js file
